build/commands/lib/util.js

```diff
@@ -879,6 +881,32 @@ const util = {
     const excludeFileName = util.getGitInfoExcludeFileName(repoDir, true)
     fs.appendFileSync(excludeFileName, '\n' + exclusion)
   },
+
+  iosCreateXCFrameworks: (buildConfig = config.defaultBuildConfig,
```

Comment:
please move this into a separate file into `brave/build/commands/scripts` dir (`ios.js` or `iosCommands.js` for ex.).

Reply:
Fixed here: https://github.com/brave/brave-core/pull/21553/commits/75eff03e08eac9ca46a5b2deed1445359c57a58c